api/src/main/java/org/openmrs/util/DatabaseUpdater.java

```diff
@@ -199,12 +199,11 @@ public void visit(ChangeSet changeSet, DatabaseChangeLog databaseChangeLog, Data
 		log.debug("Setting up liquibase object to run changelog: " + changeLogFile);
 		Liquibase liquibase = getLiquibase(changeLogFile, cl);
 		int numChangeSetsToRun = liquibase.listUnrunChangeSets(contexts).size();
-		Database database = null;
-		LockService lockHandler = null;
+		Database database = null;. lockHandler = null;
```

Comment:
@georgejunior You have a compilation issue in this line.  you need to remove **. lockHandler = null;** and add LockService lockHandler = null;

Reply:
okay let me try that

Reply:
> @georgejunior You have a compilation issue in this line. you need to remove **. lockHandler = null;** and add LockService lockHandler = null;

i am still getting a build failure

Reply:
@georgejunior can you please push the code so we can have a look ?

Reply:
@georgejunior I can still see the line that i've asked you to change. You should remove the line marked in red color and keep it as its marked by blue color.

![image](https://user-images.githubusercontent.com/3784200/62333414-a1578280-b4c3-11e9-92c2-e20e02a87a1f.png)